base.py

```diff
@@ -426,14 +467,41 @@ class MBDynSettingsObject(bpy.types.PropertyGroup):
             update = update_label
             )
 
-    # String representing MBDyn's node string label assigned to the object. 
+    # String representing MBDyn's node string label assigned to the object.
     # Non-"not assigned" only if a .lab file with correct syntax is found
     string_label = StringProperty(
             name = "MBDyn's node or joint string label",
             description = "String label of MBDyn's node assigned to the object (if present)",
             default = "not assigned"
             )
 
+    #String representing path of the output directory
```

Comment:
Why place all this properties in `MBDynSettingsObject`? This way, each object will have its own `dir_path` and  `install_path` property. Furthermore, if the scene is empty, there's no entity that holds their value. 

Move them in `MBDynSettingsScene`, please.